example/lib/attribution.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+import 'package:maplibre_gl/mapbox_gl.dart';
+
+import 'page.dart';
+
+class AttributionPage extends ExamplePage {
+  const AttributionPage({super.key})
+      : super(const Icon(Icons.thumb_up), 'Attribution');
+
+  @override
+  Widget build(BuildContext context) {
+    return const AttributionBody();
+  }
+}
+
+class AttributionBody extends StatefulWidget {
+  const AttributionBody({super.key});
+
+  @override
+  State<AttributionBody> createState() => _AttributionBodyState();
+}
+
+class _AttributionBodyState extends State<AttributionBody> {
+  AttributionButtonPosition? attributionButtonPosition;
+  bool useDefaultAttributionPosition = false;
```

Comment:
```suggestion
  bool useDefaultAttributionPosition = true;
```
small request: could we show the attribution by default here, to set a good example?

Reply:
Sure. I've also changed the order of the example button to have the "default" first. 
The gif in the PR is updated.